src/main/scala/go/Hash.scala

```diff
@@ -0,0 +1,1239 @@
+    "5bd3613d2ee222fdb2ebc33394603af6",
+    "c928bda035f8c39d84b3b1b6fa01cb1a",
+    "29e8eeca9b09c735e16b42d53a9ecf6e",
+    "dc35bba3f78ed4ee27b24383307c7a88",
+    "1753c5b3ef820c8101635963de0d35f7",
+    "ef3368ab56565ae7d5667fb942e52b2f",
+    "ebf48bd35c4ead40958faa7325a97d06",
+    "529b39d015e4975570433ecb73b7ad92",
+    "697ea70620e987514ab8a46b4c36eaaa",
+    "ebcabe3d4cbc02121869111d7c4fe1a8",
+    "2f424e669d2a7f1bfcd46428ff1e2a1e",
+    "6ae47a9c22302f58b65e4b536bfa3559",
+    "83f9a7b574523121180b7095853d37f3",
+    "77b6860daa7a39d5bb0226e8c1543063",
+    "1611e306f167e512708d753a0092df11",
+    "2d78f39e1adbaa9d9992315a0c7adfe5",
+    "1aada836dedb3ba7e12508294de8e35e",
+    "f37991753c7df55849960c597d119ace",
+    "e80840e623a19d0857b7ccbb21f74d1c",
+    "31d71dce64b2c310ca3c7f8d050c44ba",
+    "f165b587df8981908f50a115834e5414",
+    "a57e6339dd2cf3a077568e6e61516b92",
+    "1ef6e6dbb1961ec9d153e6cf8d1984ea",
+    "70cc73d90bc26e2413099942ab633504",
+    "e21a6b35df0c3ad7946c73529a2f3850",
+    "003a93d8b28069623d1adc27d706b921",
+    "1c99ded33cb890a1994b8bd260c3fad2",
+    "cf3145de0add4289f4cf0c83cace7fe4",
+    "d0e4427a5514fb7254807a18b6952e27",
+    "77c621cc9fb3a483e2a1aff40d08315c",
+    "67a34dac4356550b47ec43ffbc092584",
+    "1d6dc0ee61ce803e6a2ad922a69a13e9",
+    "c6284b653d38e96a49b572c7942027d5",
+    "803f5fb0d2f97fae08c2e9271dc91e69",
+    "b183ccc9e73df9ed088dfad983bb7913",
+    "fdeef11602d6b44390a852cacfc0adeb",
+    "1b0ce4198b3801a6c8ce065f15fe38f5",
+    "366a2a47eb189287009f592bd8d06661",
+    "0622f736d98ed771f6a5d2bfc99deba7",
+    "25f9d3eaf92c9e91fe9714e012c0856d",
+    "ecd1a8444d42907ab8ccb16b4c0ac225",
+    "b936e514ed5d055b1ab045ed7e62394d",
+    "41731f54f27b958194d7d5ec0b0843d5",
+    "b468ce588498860cc9ebf774e6329d72",
+    "3222cdd8ab559c74f762ba780eed53f1",
+    "e9c1c9c8a0d731ba57c358cdcfaa4076",
+    "f225fcc6274bf6b43d518a67ad59310f",
+    "50ee4abb80367b883c11ae32fec8a491",
+    "8e98acbad40096b73760e466d631dbaa",
+    "a63f45856b35535b0bbb8239d1e82cd3",
+    "51d8eda93f812cebb08fd2c7c7ea5b1e",
+    "eaf0cbd47bb6b9e036dbfc22b38690ec",
+    "00c7eda1ad6f049b70c19d004245077e",
+    "c389f7b07608a9d8d18ec527accc8471",
+    "82c2777aac05f18c04b55e5902edcd4a",
+    "1971b57deeac5a1ac3c8d5d6ffe58721",
+    "b5eee6d125c97169bfa205cc4e1120f8",
+    "864c1f7b7af5cf42e903851ac85de67f",
+    "017a2b4bd347e79b88408adcb080a9c4",
+    "5c6cd9ca4a28e228fa8a106b2aac9aab",
+    "6c89e0a23631f8f82288be444befc00f",
+    "1872617aa276a997f19f0c756d4b59b0",
+    "6323b3848a3736924b124de906928d47",
+    "274fad1638408eb76151e2d4cd13bf97",
+    "adbbb3829f67da94c221a62a16628ea2",
+    "05d11a0355eafd36c6fd5ad6d46847ae",
+    "61343e424609be2bb20f5ca1853b8623",
+    "4dd0158dd9cb6c43e32478856ebfe46d",
+    "bf706d4fc2a06b5ef87f72039c1bbdbd",
+    "6ea60105f1fb7e27daff57170b970b62",
+    "cd7299957813eb250ee10e6d59320d61",
+    "557b3167b30e164871429817f41392b7",
+    "8326332c4fd9f96d677c05299b05e087"
```

Comment:
cant remember all the details behind how many entries to have - but just to check that this has been calculated and set correctly for Go? Feel like it should be large as there are a large number of possible squares

Reply:
yes added comment